config/web.php

```diff
@@ -5,4 +5,10 @@
 /* @var array $params */
 
 return [
+    \App\BootstrapHandler::class => [
+        '__class' => \App\BootstrapHandler::class,
+        '__construct()' => ['callbacks' => [
+            [\App\TestBootstrap::class, 'bootstrap'],
```

Comment:
We can use dedicated `bootstrap.php` for callbacks and collect/merge it from packages via composer config plugin.